src/libs/actions/Report.ts

```diff
@@ -127,6 +127,14 @@ const allReports: OnyxCollection<Report> = {};
 let conciergeChatReportID: string | undefined;
 const typingWatchTimers: Record<string, NodeJS.Timeout> = {};
 
+let reportIDDeeplinkedFromOldDot: string | undefined;
+Linking.getInitialURL().then((url) => {
+    const params = new URLSearchParams(url ?? '');
+    const exitToRoute = params.get('exitTo') ?? '';
+    const {reportID} = ReportUtils.parseReportRouteParams(exitToRoute);
+    reportIDDeeplinkedFromOldDot = reportID;
+});
```

Comment:
Why wasn't this added in a `useEffect` with an empty dependency list?

Reply:
Just wanted to avoid passing this ugly parameter from a component to the `Report.addComment` and `Report.addAttachment` functions. Do we have a best practice or a guideline related to this? 👀

Reply:
I don't know of any. I guess this is fine as it is then.